charms/worker/k8s/src/charm.py

```diff
@@ -1340,6 +1358,43 @@ def _on_refresh_certs_action(self, event: ops.ActionEvent):
             except (InvalidResponseError, K8sdConnectionError) as e:
                 event.fail(f"Failed to refresh certificates: {e}")
 
+    @on_error(
+        ops.BlockedStatus("Cannot change bootstrap config after bootstrap. Check logs."),
+        BootstrapConfigChangeError,
+    )
```

Comment:
perfect... the user configured something and that's not valid to change.  Tell them to fix their configuration